src/source/src/source_desc.rs

```diff
@@ -102,7 +102,7 @@ impl SourceDescBuilder {
     pub fn build(mut self) -> Result<SourceDesc> {
         let columns = self.column_catalogs_to_source_column_descs();
 
-        let psrser_config = SpecificParserConfig::new(&self.source_info, &self.properties)?;
+        let psrser_config = SpecificParserConfig::new(&self.source_info, &self.properties, None)?;
```

Comment:
Why it's `None` here? Is it correct?

Reply:
Since for now aws auth options are only used by frontend to fetch schema, executors do not need them (and cannot get them because only merged props are persisted).